notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb

```diff
@@ -0,0 +1,316 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loading Models into Elasticsearch from Hugging Face\n",
+    "\n",
+    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/elastic/elasticsearch-labs/blob/main/colab-notebooks-examples/integrations/hugging-face/loading-model-from-hugging-face.ipynb\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
```

Comment:
I'm not 100% sure if this link is correct, so please correct me if its wrong.

Reply:
Looks correct to me!